graphistry/models/gfql/types/predicates.py

```diff
@@ -0,0 +1,39 @@
+"""
+Type definitions for GFQL predicate inputs
+
+Defines what types users can pass to predicates.
+Implementation details (like what gets stored internally) belong in compute/predicates.
+"""
+
+from typing import Union, Any, Dict
+import numpy as np
+
+from .temporal import NativeTemporal, TemporalWire
+from .numeric import NativeNumeric
+
+
+# ============= Basic Types =============
+# Simple scalar types
+
+BasicScalar = Union[int, float, str, np.number, None]
+
+
+# ============= Predicate Input Types =============
+# What users can provide to each predicate type
+
+# Comparison predicates (GT, LT, GE, LE, EQ, NE) - strict, no strings
+ComparisonInput = Union[
```

Comment:
afaict these are just helpers for the predicates, so can move to the predicate files?